job-service/src/main/java/com/hpe/caf/services/job/api/ApiExceptionMapper.java

```diff
@@ -41,13 +44,20 @@
     @Override
     public Response toResponse(Exception exception) {
         final Response.Status httpStatus;
-        if (exception instanceof BadRequestException) {
+        if (exception instanceof BadRequestException ||
+            exception instanceof UnrecognizedPropertyException
+        ) {
             httpStatus = Response.Status.BAD_REQUEST;
         } else if (exception instanceof NotFoundException) {
             httpStatus = Response.Status.NOT_FOUND;
         } else if (exception instanceof ForbiddenException) {
             httpStatus = Response.Status.FORBIDDEN;
-        } else if (exception instanceof ServiceUnavailableException) {
+        } else if (
+            exception instanceof ServiceUnavailableException ||
+            exception instanceof AlreadyClosedException ||
+            exception instanceof TimeoutException ||
+            exception instanceof IOException
```

Comment:
How come you have to add all these other exceptions when you're already updated `JobsPut.java` to throw a `ServiceUnavailableException`?

Reply:
I added those as in createOrUpdateJob(), we contact the database several times before reaching [the part where we intent to create a job](https://github.com/JobService/job-service/blob/bea9a358ea4767e812d054a0f130d67ee2f998c1/job-service/src/main/java/com/hpe/caf/services/job/api/JobsPut.java#L208)
So, I just wanted to cover the different disconnection exceptions. From my understanding, the idea of that 
 ApiExceptionMapper is to categorize all exceptions likely to return a Service Unavailable code.